apps/biobin/resources/usr/bin/biobin-permute-collate.py

```diff
@@ -5,74 +5,66 @@
 
 
 if (len(sys.argv) < 3) or ("%d" not in sys.argv[2]):
-	print "usage: %s <baseline> <pathformat> [permutations]" % (sys.argv[0],)
-	print "\t<baseline> is the unpermuted summary.tsv file"
-	print "\t<pathformat> is the path to the permuted summary.tsv files,"
-	print "\t\twhich must contain \"%%d\" in place of the permutation number"
-	print "\t[permutations] is the number of permutations to scan; default=1000"
-	sys.exit(2)
+    print "usage: %s <baseline> <pathformat> [permutations]" % (sys.argv[0],)
+    print "\t<baseline> is the unpermuted summary.tsv file"
+    print "\t<pathformat> is the path to the permuted summary.tsv files,"
+    print "\t\twhich must contain \"%%d\" in place of the permutation number"
+    print "\t[permutations] is the number of permutations to scan; default=1000"
+    sys.exit(2)
 
 
 pMin = 1
 pMax = 1000
 if len(sys.argv) > 3:
-	pRange = sys.argv[3].split('-',1)
-	if len(pRange) > 1:
-		pMin = int(pRange[0])
-		pMax = int(pRange[1])
-	else:
-		pMax = int(pRange[0])
-#if arg2
+    pRange = sys.argv[3].split('-',1)
+    if len(pRange) > 1:
+        pMin = int(pRange[0])
+        pMax = int(pRange[1])
+    else:
+        pMax = int(pRange[0])
 
 results = list()
```

Comment:
Should always use [] and {} for list/dict creation

https://stackoverflow.com/questions/5790860/and-vs-list-and-dict-which-is-better